### PR TITLE
Go live 28/8

### PR DESCRIPTION
**Pull request by ITViking**
#### What does this PR do?
It add's primary and secondary routes, so the libraries can go live
The libraries are the ones mentioned here: https://reload.zulipchat.com/#narrow/stream/435665-DDF---Drift.2B/topic/Udrulning.20uge.2035.20-.20en.20lille.20.C3.A6ndring/near/465545830

#### Should this be tested by the reviewer and how?
Read it through test the urls are correct in your browser

#### What are the relevant tickets?
https://reload.atlassian.net/jira/software/c/projects/DDFDRIFT/boards/464?selectedIssue=DDFDRIFT-201

### CODE REVIEW

File: infrastructure/environments/dplplat01/sites.yaml [hunk 1]
```diff
@@ -634,6 +650,10 @@ sites:
     name: "Rødovre Bibliotek"
     description: "The library site for Rødovre"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOLyDhYdpnunBj8EZNR4fE6lN4vc/+RVVB5n2ei/40xt"
+    primary-domain: www.rdb.dk
```
Comment (spaceo) [inline infrastructure/environments/dplplat01/sites.yaml:653]: Sure about the order of primary/secondary domains here?

Comment (ITViking) [reply infrastructure/environments/dplplat01/sites.yaml:653]: yep

File: infrastructure/environments/dplplat01/sites.yaml [hunk 2]
```diff
@@ -272,6 +280,10 @@ sites:
     name: "Gribskov Biblioteker"
     description: "The library site for Gribskov"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIIK40x6Y/UvhxDaW+McgqIeY2phSuuadNxuG/FGnSY70"
+    primary-domain: www.gribskovbib.dk
```
[the remark this answers is not in the document]
Comment (ITViking) [reply infrastructure/environments/dplplat01/sites.yaml:283]: yep

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**